charts/argocd-image-updater/values.yaml

```diff
@@ -44,6 +44,14 @@ extraEnv: []
   # - name: AWS_REGION
   #   value: "us-west-1"
 
+# -- envFrom to pass to argocd-image-updater
+# @default -- `[]` (See [values.yaml])
```

Comment:
This line not needed IMO

Reply:
@jmeridth I removed it.
Initially I compared it to this: https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L2682